Bozidar_Yukopila/Tarea4/index.js

```diff
@@ -0,0 +1,75 @@
+let numvar = "";
```

Comment:
@Bozidary24  , research about the use of const, let and var and refactor accordingly

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/